examples/start-shadcn-example/scripts/build-registry.sh

```
@@ -0,0 +1,59 @@
+
+for target in "${TARGETS[@]}"; do
+  target_components="$target/src/components"
+  target_lib="$target/src/lib"
+
+  echo "→ syncing $target"
+
+  mkdir -p "$target_components" "$target_lib"
+
+  # Prune stale mirrors so renames/deletes propagate.
+  for dir in "${COMPONENT_DIRS[@]}"; do
+    rm -rf "$target_components/$dir"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)